src/test/java/net/sf/jabref/logic/citationstyle/CitationStyleTest.java

```diff
@@ -0,0 +1,14 @@
+package net.sf.jabref.logic.citationstyle;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class CitationStyleTest {
+
+    @Test
+    public void getDefault() throws Exception {
+        Assert.assertNotNull(CitationStyle.getDefault());
+    }
+
```

Comment:
I added a Test to check the generated citation with the default citation style on the test bibEntry.